symplyphysics/definitions/wavelength_is_gap_between_peaks.py

```diff
@@ -0,0 +1,32 @@
+from symplyphysics import (
+    symbols, Eq, pretty, Quantity, units, solve,
+    validate_input, validate_output, expr_to_quantity, SI
+)
+
+# Description
```

Comment:
Move it to laws and use some better name, eg `wavelength_from_velocity_and_period`